go/example.go

```diff
@@ -187,25 +186,23 @@ func main() {
 		defer rec.Release()
 		log.Println(rec)
 	}
+	return nil
 }
 
 func setSessionOptions(ctx context.Context, client flight.Client, projectID string) error {
 	projectIdSessionOption, err := flight.NewSessionOptionValue(projectID)
 	if err != nil {
 		return fmt.Errorf("failed to create session option: %v", err)
 	}
-
 	sessionOptionsRequest := flight.SetSessionOptionsRequest{
 		SessionOptions: map[string]*flight.SessionOptionValue{
 			"project_id": &projectIdSessionOption,
 		},
 	}
-
 	_, err = client.SetSessionOptions(ctx, &sessionOptionsRequest)
 	if err != nil {
-		return fmt.Errorf("failed to set session options: %v", err)
```

Comment:
nit, but previous error message (starting with "failed to" was consistent with other ones in this file)